meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -196,6 +196,20 @@ function runPreHacks {
         sed --in-place "s|curl --silent --header \"User-Agent:\" --compressed|wget -qO-|" $script_path
         sed --in-place "s|curl -s --compressed|wget -qO-|" $script_path
     fi
+
+    # Earlier BeagleBone releases might fail to update due to memory pressure.
+    # The fix for this was released in v1.24.0, for versions below that apply
+    # changes manually
+    if [ "$SLUG" == "beaglebone-black" ] ||
+       [ "$SLUG" == "beaglebone-green" ] ||
+       [ "$SLUG" == "beaglebone-green-wifi" ]; then
```

Comment:
@willnewton yeah, that's true, thanks! it got merged before I could address this, probably just refactor later?